export default async function ValidateToken(req: any, res: any) {
  try {
    // const res = await
  } catch (error) {
    res.status(500).json({ error: 'Woops! something went wrong' })
  }
}
